themes/tiddlywiki/vanilla/base.tid

```diff
@@ -2186,3 +2186,7 @@ body.tc-dirty span.tc-dirty-indicator, body.tc-dirty span.tc-dirty-indicator svg
 	background: #f00;
 	color: #fff;
 }
+
+.tc-keyboard {
+	display: inline;
```

Comment:
> for some reason

It's pretty obvious: we need to DIV so that we've got somewhere to attach the keyboard event handlers.